libs/designer-ui/src/lib/workflow/languageservice/templatefunctions.ts

```diff
@@ -2942,6 +2954,28 @@ export const FunctionGroupDefinitions: FunctionGroupDefinition[] = [
         ],
         isAdvanced: false,
       },
+      {
+        name: 'dateDifference',
+        defaultSignature: 'dateDifference(startTimestamp, endTimestamp)',
+        description: Resources.TOKEN_FUNCTION_DATETIME_DATEDIFFERENCE,
+        signatures: [
+          {
+            definition: `convertTimeZone(timestamp: string, sourceTimeZone: string, destinationTimeZone: string, format?: string)`,
```

Comment:
is this definition correct?

Reply:
oops you're right, i copied the wrong one, will fix